src/views/migrationpolicies/list/utils/constants.ts

```diff
@@ -0,0 +1,3 @@
+import { MigrationPolicyModelRef } from '@kubevirt-ui/kubevirt-api/console';
+
+export const mirationPoliciesPageBaseURL = `/k8s/cluster/${MigrationPolicyModelRef}`;
```

Comment:
Maybe better
`migrationPoliciesPageBaseURL` instead of
`mirationPoliciesPageBaseURL`? (just a little typo)

Reply:
nice catch fixed